src/reynir_correct/errtokenizer.py

```diff
@@ -2402,6 +2425,32 @@ def number_error(
             at_sentence_start = False
 
 
+def late_fix_merges(
+    token_stream: Iterable[CorrectToken],
+    db: GreynirBin,
+    token_ctor: TokenCtor,
+    only_ci: bool,
+) -> Iterator[CorrectToken]:
+    """Annotate tokens where error annotation
+    has disappeared due to token merging"""
+    for token in token_stream:
+        if (
+            token.original
+            and token.txt.strip() != token.original.strip()
+            and isinstance(token, CorrectToken)
```

Comment:
Could do a simpler test: and not hasattr(token, "error")

Reply:
Done

Reply:
Had to change back; the test stopped working